control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -222,6 +243,7 @@ func Setup(ctx context.Context, opts *operator.HostedClusterConfigOperatorConfig
 		&corev1.Secret{},
 		&corev1.Service{},
 		&corev1.Endpoints{},
+		&corev1.Node{},
```

Comment:
why do we need to watch nodes?

Reply:
Ok. It's there 'cause I had the cached client without it I couldn't list nodes properly.  I'm going to get rid of it, i